### ISSUE TITLE
Focus rings have different colors depending on user agent

### ISSUE THREAD

**Issue by PrinceYauj** [freeats/freeats#260]
**Problem**
Looks like the focus rings are different depending on user agent/operational system:
![image](https://github.com/user-attachments/assets/b1b235c9-1130-4fd3-a7e3-fa14ee5837ab)
![image](https://github.com/user-attachments/assets/cbaaf69d-18b0-4aac-b541-de8162a3419d)
![image](https://github.com/user-attachments/assets/8bb28358-0774-4f9e-aa5d-961dd201eca1)

**Solution**
Focus rings should have a tabler's style for all users:
![image](https://github.com/user-attachments/assets/b7b14ca0-22db-4088-9729-1e71aad1f960)

As a part of this issue, fix focus ring around the logo:
![image](https://github.com/user-attachments/assets/83042713-8cd8-4f46-ae1a-666c86066287)

**Comment by PrinceYauj** [comment]
duplicate https://github.com/freeats/freeats/issues/258
